client/mass/charts.js

```diff
@@ -17,6 +17,9 @@ class MassCharts {
 			tip: new Menu({ padding: '0px' }),
 			tooltip: new Menu({ padding: '4px' })
 		}
+		this.opts.holder.on('mouseleave', () => {
+			this.dom.tooltip.hide()
```

Comment:
it only hide tooltip on leaving the chart button tray. it instead should hide it on leaving the chart button (that carry the tooltip)